src/app/related-search/related-search.component.ts

```diff
@@ -52,7 +52,18 @@ export class RelatedSearchComponent implements OnInit {
       }
     });
   }
-
+  getQuery(q) {
+    return {query: q.toString(),
+      verify: false,
+      nav: 'filetype,protocol,hosts,authors,collections,namespace,topics,date',
+      start: 0,
+      indexof: 'off',
+      meanCount: '5',
```

Comment:
do we actually need the meancount? I don't find it used anywhere in the component